### DISCUSSION TITLE
Delete time series with multiple wildcard match expressions.

### DISCUSSION THREAD

**Discussion by raz08** [prometheus/prometheus#12899] [General]
hello, 
Is it possible to delete time series with multiple match queries with wild card? I have tried deleing series with below URL but is it not working.

`"http://$pod_ip:9090/api/v1/admin/tsdb/delete_series?match[]={job=\"test\",pod=~\".*\",handler=~\".*\"}&end=$output_time"`

If I send this query it deletes all time series in job test irrespective of pod and handler. 

Expected results: Should delete time series in job test which has any pod, handler label.

I have tried multiple combinations but didn't help. 

```
/delete_series?match[]={job=\"test\"}&match[]={pod=~\".*\"}&match[]={handler=~\".*\"}&end=$output_time

/delete_series?match[]={job=\"test\",pod=~\".*\",handler=~\".*\"}&end=$output_time
```

First thing is it supported? deleting time series with multiple wildcard match exprs ?. any help is appreciated.

**Comment by RohanKumarMainali** [comment]
try to explore DeletePartialMatch function
